documentation/library_guide/api_for_sycl_kernels/tested_standard_cpp_api.rst

```diff
@@ -453,18 +453,12 @@ C++ Standard API                      libstdc++  libc++     MSVC
 These tests were done for the following versions of the standard C++ library:
 
 ============================================= =============================================
-libstdc++ (GNU)                               Provided with GCC*-7.5.0, GCC*-9.3.0
+libstdc++ (GNU)                               Provided with GCC*-8.4.0, GCC*-9.3.0,
+                                              GCC*-11.4.0, GCC*-13.2.0
 --------------------------------------------- ---------------------------------------------
-libc++ (LLVM)                                 Provided with Clang*-11.0
+libc++ (LLVM)                                 Provided with Clang*-11.0, Clang*-14.0,
+                                              Clang*-17.0, Clang*-18.1
```

Comment:
@dcbenito Are the asterisks really necessary for the compiler names? I do not think these are trademarks, really.

Reply:
@timmiesmith I also think that hyphens between the name and the version are rather unusual, though I do not mind it either.

Reply:
An asterisk is only needed on the first instance of a non-Intel product, so we only need a single asterisk. The hyphens, in my opinion, are not needed, as this is just a version number and we don't follow that format for any other version numbers.

Reply:
I've removed the hyphens and limited the asterisk to the first use as suggested.

Reply:
LGTM.